home/service/glossary.py

```diff
@@ -0,0 +1,46 @@
+from data_platform_catalogue.search_types import MultiSelectFilter, ResultType
+from django.core.exceptions import ObjectDoesNotExist
+from copy import deepcopy
+from itertools import groupby
+
+from .base import GenericService
+
+
+class GlossaryService(GenericService):
+    def __init__(self):
+        # Can we put the client instantiation in the base class?
```

Comment:
Sounds like a good idea to me. Either that or create a new base class especially for services that uses the catalogue.